### ISSUE TITLE
Latest tag doesn't appear to be latest from Flysystem

### ISSUE THREAD

**Issue by bmidget** [thephpleague/flysystem-aws-s3-v3#283]
On the flysytem repo, see commit https://github.com/thephpleague/flysystem/commit/ca8bb7eff69fce4034fbc72cf7e6c6af0948fe7e

But the tarball from the latest tag `3.0.6` on this composer package clearly does not have this commit applied.

**Comment by frankdejonge** [comment]
This is the zip: https://github.com/thephpleague/flysystem-async-aws-s3/archive/refs/tags/3.0.6.zip

**Comment by frankdejonge** [comment]
Btw, are you sure you're looking at the right package? The commit was for async-aws-s3, not this subsplit, which is the regular aws s3.

**Comment by bmidget** [comment]
You're correct. I saw the commit for the async package and was confused why it wasn't applied to this package.

But the same issue that you fixed on the async library still exists on this one. The `EXTRA_METADATA_FIELDS` constant is declared private and therefore this class cannot properly be extended.

Therefore this issue should probably be closed and ignored, but the `private` constant issue still exists here.
